app/controllers/admin/users_controller.rb

```diff
@@ -80,13 +80,15 @@ def update
       existing_health_roles = @user.health_roles.to_a
       begin
         User.transaction do
+          changing_to_acls = params[:user][:permission_context] == 'acls' && @user.permission_context != params[:user][:permission_context]
```

Comment:
Let's pull these out into methods, for now they can just live in the controller.  
```
private def changing_to_acls?
  params[:user][:permission_context] == 'acls' && @user.permission_context != params[:user][:permission_context]
end

private def user_using_or_changing_to_acls?
  @user.using_acls? || changing_to_acls
end
```